run_test.py

```diff
@@ -31,12 +31,11 @@
         script = './test.sh'
 
     elif args.test == 'py3':
-        # TODO(unno): Fix this line after protobuf 3 is released
         conf = {
             'base': 'ubuntu14_py3',
             'cuda': 'cuda70',
             'cudnn': 'cudnn4',
-            'requires': ['setuptools', 'cython', 'numpy<1.11', 'protobuf==3.0.0b3'],
+            'requires': ['setuptools', 'cython', 'numpy<1.11', 'protobuf<3'],
```

Comment:
OK, I'll write it in README. As you know, i'm making a more detailed document now.